builder/bootloader.py

```diff
@@ -64,9 +100,13 @@ def get_suitable_optiboot_binary(framework_dir, board_config):
 if core in ("MiniCore", "MegaCore", "MightyCore", "MajorCore"):
     if not isfile(bootloader_path):
         bootloader_path = get_suitable_optiboot_binary(framework_dir, board)
+elif core == "MicroCore":
+    if not isfile(bootloader_path):
+        bootloader_path = get_suitable_urboot_binary(framework_dir, board)
 else:
     if not isfile(bootloader_path):
         bootloader_path = join(framework_dir, "bootloaders", bootloader_path)
+        print(bootloader_path)
```

Comment:
Is this leftover debug code? If no, maybe a bit more info added? Something like:
```python
"Using bootloader image: `%s`" % bootloader_path
```

Reply:
It doesn't hurt, and it makes the whole process a little more transparent. BTW what's the deal with the \`` quotes in your example? Are these necessary? And how does they differ from the single quotes ' ' ?

Reply:
> BTW what's the deal with the `` quotes in your example? Are these necessary? And how does they differ from the single quotes ' ' ?

Nothing special, just a habit came from writing markdown.